Source/BugsnagNotifier.m

```diff
@@ -287,6 +290,17 @@ - (void) start {
 #endif
 }
 
+- (void)setupConnectivityListener {
+    NSURL *url = self.configuration.notifyURL;
+    self.networkReachable = [[BSGConnectivity alloc] initWithURL:url];
+    [self.networkReachable startWatchingConnectivity];
```

Comment:
Related to the other comment, since `BSGConnectivity` observes reachability on a different queue. There could be a timing problem if reachability becomes connected after starting watching and before `connectivityChangeBlock` is set below. Since `connectivityChangeBlock` will be nil it would cause a crash.